### PR TITLE
ghostscript-fonts: Fix URL.

### PR DESCRIPTION
**Pull request by RemiLacroix-IDRIS**

### CODE REVIEW

**Comment by RemiLacroix-IDRIS** [comment]
Am I supposed to break the url line to make flake8 happy? It doesn't fell like this has been done in any other packages.

**Comment by tldahlgren** [comment]
> Am I supposed to break the url line to make flake8 happy? It doesn't fell like this has been done and any other packages.

Afraid so.  Looks like 88 characters is the line limit.

**Comment by RemiLacroix-IDRIS** [comment]
Not sure what the canonical way of doing this in this case.

I could use string concatenation:
```python
    sourceforge_mirror_path = "gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/" +
                              "ghostscript-fonts-std-8.11.tar.gz"
```
but maybe there is a better solution using a `join_url` function (not sure it that exists).

**Comment by tldahlgren** [comment]
> Not sure what the canonical way of doing this in this case.
> 
> I could use string concatenation:
> 
> ```python
>     sourceforge_mirror_path = "gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/" +
>                               "ghostscript-fonts-std-8.11.tar.gz"
> ```
> 
> but maybe there is a better solution using a `join_url` function (not sure it that exists).

Most people tend to use some form of concatenation, e.g.

```
    sourceforge_mirror_path = ("gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/"
                               "ghostscript-fonts-std-8.11.tar.gz")
```

There is a URL `join` but it is overkill for this situation and we don't expose it to packages.

**Review by tldahlgren — COMMENTED** [review]
Confirmed the sha256 is still correct with the new URL.

**Comment by RemiLacroix-IDRIS** [comment]
> Most people tend to use some form of concatenation

Ok, I went with some basic string concatenation then.

**Comment by RemiLacroix-IDRIS** [comment]
Seems like style tests failed but I don't think that's on me!

**Comment by tldahlgren** [comment]
Closing and re-opening to force re-running checks (due to bootstrap issue affecting style tests).

**Review by tldahlgren — DISMISSED** [review]

File: var/spack/repos/builtin/packages/ghostscript-fonts/package.py [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 
 
-class GhostscriptFonts(Package):
+class GhostscriptFonts(Package, SourceforgePackage):
     """Ghostscript Fonts"""
 
     homepage = "https://ghostscript.com/"
-    url = "https://www.imagemagick.org/download/delegates/ghostscript-fonts-std-8.11.tar.gz"
+    sourceforge_mirror_path = "gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/" + 
+                              "ghostscript-fonts-std-8.11.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/ghostscript-fonts/package.py]: This is invalid syntax. I think people use more often parenthesis, but if not you need `\` after the `+` sign to have a multiline expression.

Comment (RemiLacroix-IDRIS) [reply var/spack/repos/builtin/packages/ghostscript-fonts/package.py]: Alright, I should definitely have tested that...

**Review by alalazo — APPROVED** [review]

File: var/spack/repos/builtin/packages/ghostscript-fonts/package.py (hunk 1 above)
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/ghostscript-fonts/package.py]: 🤦

**Pull request merged**